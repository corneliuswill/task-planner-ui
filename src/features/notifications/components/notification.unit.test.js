import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Notification from './notification';

describe('<Notification/>', () => {
    it('component renders correctly', () => {
        const wrapper = shallow(<Notification message='Hello World!'/>);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});