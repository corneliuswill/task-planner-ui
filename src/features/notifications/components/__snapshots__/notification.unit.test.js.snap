// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Notification/> component renders correctly 1`] = `
<styled.div
  className="notification-container"
  onClick={[Function]}
  style={
    Object {
      "backgroundColor": "#CFE2FF",
      "borderColor": "#D6D4FE",
      "color": "#084298",
    }
  }
>
  <p
    className="message"
  >
    <span
      className="font-700"
    >
      INFO:
    </span>
     
  </p>
</styled.div>
`;

exports[`<Notification/> should render text message correctly 1`] = `
<styled.div
  className="notification-container"
  onClick={[Function]}
  style={
    Object {
      "backgroundColor": "#CFE2FF",
      "borderColor": "#D6D4FE",
      "color": "#084298",
    }
  }
>
  <p
    className="message"
  >
    <span
      className="font-700"
    >
      INFO:
    </span>
     
    Hello World!
  </p>
</styled.div>
`;
